lib/components/tabs.vue

```diff
@@ -146,28 +142,28 @@
              * Move to next tab
              */
             nextTab() {
-                this.setTab(this.currentTab, false, 1);
+                this.setTab(this.currentTab + 1, false, 1);
             },
 
             /**
              * Move to previous tab
              */
             previousTab() {
-                this.setTab(this.currentTab, false, -1);
+                this.setTab(this.currentTab - 1, false, -1);
             },
 
             /**
              * Set active tab on the tabs collection and the child 'tab' component
              */
-            setTab(index, force, offset) {
-                offset = offset || 0;
+            setTab(index, force, direction) {
+                direction = this.sign(direction || 0);
 
                 // Prevent setting same tab!
-                if (!force && (index + offset) === this.currentTab) {
+                if (!force && index === this.currentTab) {
```

Comment:
and we have this expression at fact:
`if (!force && (this.currentTab+[1|-1] === this.currentTab)) {`